java/spacewalk-java.changes.meaksh.Manager-4.3-prevent-issues-on-duplicate-ips-bsc1226461

```diff
@@ -0,0 +1 @@
+- Autoinstallation: prevent issues with duplicate IP address due to some networks (bsc#1226461)
```

Comment:
I would rephrase this slightly to make it clearer. "due to some networks" is hard to understand IMO.

```suggestion
- Autoinstallation: prevent "duplicate IP address" issues cause by container networks (bsc#1226461)
```